.github/workflows/build_for_riscv.yml

```
@@ -58,33 +58,169 @@
+      uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
+      with:
+        name: wasmedge-riscv64-artifacts
+        path: artifacts/
+        retention-days: 1
+  
+  test_riscv64:
+    name: Test RISC-V64 in QEMU
+    runs-on: ubuntu-latest
+    needs: cross_compile_riscv64
+    steps:
+    - name: Harden the runner (Audit all outbound calls)
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)